src/Interfaces/Options/Particles/Shape/IShapeValues.ts

```diff
@@ -0,0 +1,3 @@
+export interface IShapeValues {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty interface is equivalent to `{}`.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)